lib/node_modules/@stdlib/blas/base/dnrm2/test/test.ndarray.js

```diff
@@ -53,6 +84,31 @@ tape( 'the function calculates the L2-norm of a vector', function test( t ) {
 	z = dnrm2( x.length, x, 1, 0 );
 	t.strictEqual( z, 4.0, 'returns expected value' );
 
+	x = new Float64Array([ 1e150, 1e150, 1e150, 1e150 ]);
+
+	z = dnrm2( x.length, x, 1, 0 );
+	t.strictEqual( z, 2.0e+150, 'returns expected value' );
+
+	x = new Float64Array([ 1e-155, 1e-155, 1e-155, 1e-155 ]);
+
+	z = dnrm2( x.length, x, 1, 0 );
+	t.strictEqual( z, 2.0e-155, 'returns expected value' );
+
+	x = new Float64Array([ 1e150, 1e50, 1e150, 1e50 ]);
+
+	z = dnrm2( x.length, x, 1, 0 );
+	isApprox( t, z, 1.41421e150, 2.0 );
+
+	x = new Float64Array([ 1e-155, 1e50, 1e-155, 1e50 ]);
+
+	z = dnrm2( x.length, x, 1, 0 );
+	isApprox( t, z, 1.41421e50, 2.0 );
```

Comment:
Why are you using incomplete decimal values? Here you go only to 5 decimals. If you extend to full precision, can you decrease your test tolerance? Will the results equal the exact expected result?

Reply:
@ShabiShett07 This question still remains.